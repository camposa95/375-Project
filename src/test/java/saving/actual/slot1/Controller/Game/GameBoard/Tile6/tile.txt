TerrainType: HILLS
DieNumber: 10
HasRobber: false
VertexIDs: [13, 14, 15, 23, 24, 25]
