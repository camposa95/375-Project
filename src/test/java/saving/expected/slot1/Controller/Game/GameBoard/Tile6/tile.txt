TerrainType: HILLS
DieNumber: 10
HasRobber: false
